templates/Amcache.template

```diff
@@ -0,0 +1,96 @@
+name: Windows.Detection.Amcache
+author: Zach Stanford - @svch0st
+description: |
+   Bulk indicator hunt over Amcache
+
+   This artifact is automatically generated by DetectRaptor.
+
+reference:
+  - https://github.com/mgreen27/DetectRaptor
+
+type: CLIENT
+resources:
+  timeout: 9000
+
+parameters:
+   - name: DateAfter
+     type: timestamp
+     description: "search for events after this date. YYYY-MM-DDTmm:hh:ssZ"
+   - name: DateBefore
+     type: timestamp
+     description: "search for events before this date. YYYY-MM-DDTmm:hh:ssZ"
+   - name: CriticalityLevel
+     default: .
+     description: "Only use detections matching this regex (values can be Critical,High,Medium,Low)"
+   - name: IOCs
+     type: csv
+     default: |
+%(ioc)s
+
+sources:
+  - precondition:
+      SELECT OS From info() where OS = 'windows'
+
+    query: |
+      -- firsly split IOCs by path or filename focus
+      LET all_the_things <= SELECT * FROM IOCs
+        WHERE NOT KeywordRegex = '.' AND Criticality =~ CriticalityLevel
+        GROUP BY KeywordRegex
+      
+      -- run KeywordRegex regex
+      LET hits = SELECT *
+                            FROM Artifact.CyberCX.Windows.Detection.Amcache(   
```

Comment:
This artifact wont work with the CyberCX reference. Can you change it? I can add the originalname filter to the Windows.Detection.AMCache() if you are using that?